integrations/googlecalendar/integration.definition.ts

```diff
@@ -77,3 +96,18 @@ export default new IntegrationDefinition({
     },
   },
 })
+  .extend(interfaces.listable, (self) => ({
+    item: self.event,
```

Comment:
nitpick(blocking): `self` and `this` are synonymous making this look like you are shadowing `this`, I would change the callback argument's name to something else like `definition` or `implementation` or something more descriptive